generate_def.sh

```diff
@@ -0,0 +1,9 @@
+echo "; libtommath" >tommath.def
```

Comment:
Could we please add this to the existing scripts in order to ensure that things get executed automatically and don't get out of sync?

Reply:
I merged a few scripts together and helper.pl is now the place where all generation tasks should be done. See #293

Reply:
Thanks!  I'll try to have a look this weekend

Reply:
FYI I've merged #293 so you should be able to finally do the rebase when you have the time!